apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,95 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I shall be able to undo adding a book to the reading list', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    // search for 'haskell' books
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('haskell');
+    await form.submit();
+
+    // current reading list
+    const readingListToggle = await $('[data-testing="toggle-reading-list"]');
+    await readingListToggle.click();
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement(
+        $('[data-testing="reading-list-container"]'),
+        'My Reading List'
+      )
+    );
+    const itemsBefore = await $$('.reading-list-item');
+    const closeReadingList = await $('h2 button.mat-icon-button');
+    await closeReadingList.click();
+
+    // add new book
+    const addButton = await $$('button[test-id="add-to-reading-list"][ng-reflect-disabled="false"]').first();
+    await addButton.click();
+    
+    // to be able to locate snackbar element
+    // otherwise, protractor stops to wait for snackbar to time out and disappear
+    await browser.waitForAngularEnabled(false);
```

Comment:
I think this is what is causing the flaky tests. On my machine setting this in this test (and not unsetting it afterwards) causes the `search-books.spec.ts` test to fail.